xkeysnail/input.py

```diff
@@ -17,12 +17,24 @@ def get_devices_list():
 def is_keyboard_device(device):
     """Guess the device is a keyboard or not"""
     capabilities = device.capabilities(verbose=False)
+    _KEYS = [
+                Key.SPACE,
+                Key.ENTER,
+                Key.KEY_0,
+                Key.KEY_5,
+                Key.KEY_9,
```

Comment:
Is there a reason A and Z were removed?

Reply:
If I'm not mistaken it was because it adds support for unconventional keyboards, which don't have A and Z like the PS3 BD remote.

Initially in my setup all keyboards and the PS3 BD remote started to be detected.